siac/hostdbcmd.go

```diff
@@ -227,9 +229,18 @@ func hostdbcmd() {
 				}
 			}
 
+			// Grab the score information for the active hosts.
+			hostInfo := new(api.HostdbHostsGET)
+			err := getAPI("/hostdb/hosts/"+host.PublicKeyString, hostInfo)
+			if err != nil {
+				die("Could not fetch provided host:", err)
+			}
+			score, _ := big.NewFloat(1).SetInt(hostInfo.ScoreBreakdown.Score.Big()).Float64()
+			score /= 1e70
```

Comment:
It's very magic. And the worst part is, I don't know of a good way to de-magic it, except maybe to squish it with a logarithm. But then users might get confused about just how terrible a low score is.